cmake/Modules/FindSYCLToolkit.cmake

```diff
@@ -48,44 +75,37 @@ find_file(
   NO_DEFAULT_PATH
   )
 
-# Find SYCL library fullname.
-# Don't use if(LINUX) here since this requires cmake>=3.25 and file is installed
-# and used by other projects.
-# See: https://cmake.org/cmake/help/v3.25/variable/LINUX.html
-if(CMAKE_SYSTEM_NAME MATCHES "Linux")
-  find_library(
-    SYCL_LIBRARY
-    NAMES sycl-preview
-    HINTS ${SYCL_LIBRARY_DIR}
-    NO_DEFAULT_PATH
-  )
-endif()
-# On Windows, currently there's no sycl.lib. Only sycl7.lib with version suffix,
-# where the current version of the SYCL runtime is 7.
-# Until oneAPI adds support to sycl.lib without the version suffix,
-# sycl_runtime_version needs to be hardcoded and uplifted when SYCL runtime version uplifts.
-# TODO: remove this when sycl.lib is supported on Windows
-if(WIN32)
-  set(sycl_runtime_version 7)
-  find_library(
-    SYCL_LIBRARY
-    NAMES "sycl${sycl_runtime_version}"
-    HINTS ${SYCL_LIBRARY_DIR}
-    NO_DEFAULT_PATH
-  )
-  if(SYCL_LIBRARY STREQUAL "SYCL_LIBRARY-NOTFOUND")
-    message(FATAL_ERROR "Cannot find a SYCL library on Windows")
+# By default, we use libsycl.so on Linux and sycl.lib on Windows as the SYCL library name.
+if (SYCL_COMPILER_VERSION VERSION_LESS_EQUAL 20240703)
```

Comment:
Pls. define a readable variable name for `20240703`

Reply:
By the way, have you confirmed with Intel LLVM community that the version number is stable ?

Reply:
> Pls. define a readable variable name for `20240703`

Done. Change to `PYTORCH_2.5_SYCL_TOOLKIT_VERSION`.